src/Symfony/Component/Notifier/Bridge/GoogleChat/GoogleChatTransport.php

```diff
@@ -52,10 +52,9 @@ public function __construct(
 
     public function __toString(): string
     {
-        return sprintf('googlechat://%s/%s%s',
+        return sprintf('googlechat://%s/%s',
```

Comment:
This needs to be reverted too. It's the Symfony DSN to configure this bridge, not the URL to the Google API Endpoint.

Reply:
Whoopsie, good catch, thanks! 👍